docs/src/app/navigation.jsx

```diff
@@ -10,16 +10,29 @@ import {
     colorInverse,
     colorStatic,
 } from 'react-cm-ui/styles/colorExports';
+import _ from 'lodash';
```

Comment:
I wonder if we should try and ["fix" all the Lodash imports](https://github.com/saddlebackdev/church-management/pull/3456#discussion_r424146737) in React CM UI at least, and then gradually adopt the practice as we fix bugs and do new development in HC (church-management).